en/docs/reference/guides/message-flow-in-the-api-manager-gateway.md

```diff
@@ -13,23 +13,19 @@ The Gateway of an API Manager deployment is responsible for the main business fu
 
 The handlers are request and response interceptors. The list of API handlers in WSO2 API-M are as follows:
 
--   APIMgtLatencyStatsHandler
 -   CORSRequestHandler
 -   APIAuthenticationHandler
 -   ThrottleHandler
--   APIMgtUsageHandler
 -   APIMgtGoogleAnalyticsTrackingHandler
 -   APIManagerExtensionHandler
 
 Each handler performs a specific task as mentioned in the table below. Note that some handlers are functional both at the inflow and outflow of messages.
 
 | Handler                              | Inflow                                                              | Outflow                                                         |
 |--------------------------------------|---------------------------------------------------------------------|-----------------------------------------------------------------|
-| APIMgtLatencyStatsHandler            | Publish request and response latencies, if analytics is enabled     | Publish request and response latencies, if analytics is enabled |
```

Comment:
Let's add the new handler name and details here

Reply:
will send a new PR with the new handler info.